bundles/org.openhab.binding.alarmdecoder/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="alarmdecoder"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Alarm Decoder Bridge -->
+	<bridge-type id="ipbridge">
+		<label>Alarm Decoder IP Bridge</label>
+		<description>Nu Tech Alarm Decoder IP Bridge</description>
+		<properties>
+			<property name="vendor">Nu Tech Software Solutions</property>
+		</properties>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<label>Host Name</label>
+				<description>The hostname of the Alarm Decoder device</description>
```

Comment:
```suggestion
				<label>Host Name</label>
				<context>network-address</context>
				<description>The hostname of the Alarm Decoder device</description>
```

Reply:
This parameter can be a hostname or IP address. Does the network-address context apply to both?

Reply:
that context applies to both

Reply:
Ok. Cool. Added it.